### PR TITLE
API-38465-claim-status-404-error-format

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

-  Adds a way to handle errors.messages
- Fixes a minor bug in pdf_client.

## Related issue(s)

- [API-38465](https://jira.devops.va.gov/browse/API-38465)

## Testing done

- Postman
1. run {{uri}}/services/claims/v2/veterans/1012666073V986297/526/synchronous, using the following request_body:
```               
             "mailingAddress": {
                    "addressLine1": "1234 Couch Street",
                    "addressLine2": "Apt 4",
                    "city": "Portland",
                    "state": "OR",
                    "country": "France",
                    "internationalPostalCode": "105-011"
                },
```
2. grab the claim_id from logs
3. call {{uri}}/services/claims/v1/claims/{{claim_id from first call}}
4. Verify that the 2nd call does not return a 404

## What areas of the site does it impact?
- claims controller v1
- pdf_client

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by mchristiansonVA — DISMISSED** [review]
Tested locally to establish a claim using 526 v2 /synchronous call and confirm error from EVSS docker, then call v1 claim status to confirm no errors on status call. Looks good to me!

**Comment by stiehlrod** [comment]
@RachalCassity For what info are you waiting?

File: modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb [hunk 1]
```diff
@@ -104,5 +104,9 @@ def service(auth_headers)
         EVSS::DisabilityCompensationForm::Service.new(auth_headers)
       end
     end
+
+    def get_errors(error)
+      error&.errors if error.methods.include?(:errors)
+    end
```
Comment (FonzMP) [inline modules/claims_api/app/sidekiq/claims_api/claim_establisher.rb:110]: this returns `nil` if it doesn't hit the `if`... which would mean `error&.errors` would also return `nil` - do we need this method added with the safe navigator?

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**